stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -1,46 +1,59 @@
 set(sources
-    Glibc.swift
-    Misc.c
+  Glibc.swift
+  Misc.c
 )
 
-set(output_dir "${SWIFTLIB_DIR}/glibc")
+# When cross-compiling the stdlib on Unix platforms, we'll need a separate
+# glibc for each target.
+foreach(SDK ${SWIFT_SDKS})
```

Comment:
The parent `CMakeLists.txt` [already has those very same lines](https://github.com/apple/swift/blob/2ded9fe4447553050f08a4d1afe3c92d57b93390/stdlib/public/CMakeLists.txt#L42-L44)! And to be clear: this does actually work, both on its own when building for Linux (and I assume FreeBSD), and when #1442 is rebased on top of these changes.